kroxylicious-app/pom.xml

```diff
@@ -427,5 +404,41 @@
                 </dependency>
             </dependencies>
         </profile>
+        <profile>
+            <id>enable-profiling</id>
```

Comment:
I don't think `enable-profiling` is quite the right name.     If I understand correct to enable profiling you need this profile at build time and setting the `ASYNC_PROFILER_ENABLED` env var at runtime.  So I think `include-profiling-libs` better. Also add a comment in the run script to mentions that the async libs need to be included in the built into the artefacts.

Reply:
went with `with-profiling-libs` and `-D withProfilingLibs=false` for consistency with `withAdditionalFilters`